### ISSUE TITLE
Kits23

### ISSUE THREAD

**Issue by ahmetdemirtas1** [neheller/kits23#16]
In order to classify Kits 23 and Tumor subtype, can tumor classification be made using only segmntation.nii.gz files based on the "tumor_histologic_subtype": "xxxx", key and value from the json file? Which files can I use? Or if I want to classify tumors in the kits 21 data set, which file or files should I get?

**Comment by neheller** [comment]
You are correct that the labels are in the json file, but I would probably want to use the `imaging.nii.gz` files instead of (or in conjunction with) `segmentation.nii.gz` as the model input. These imaging files are retrieved after you run follow the "Data Download" instructions on the readme.

**Comment by qianshuiyu** [comment]
Hello，segmentation.nii.gz files are generated through expert manual segmentation, or were they the result of automated segmentation methods？And how about the segmentation quality？such as Dice coefficient and Average Rank? This would help me assess the reliability and suitability of using segmented files for my study. Thank you！
